build.properties

```diff
@@ -1,11 +1,11 @@
 minecraft.version=1.7.10
-forge.version=10.13.4.1614-1.7.10
```

Comment:
I switched this to 1614 - I would like to keep it on the last version of 1.7.10 forge.